package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Aug 26 19:25:07 UTC 2015 - mfilka@suse.com
+
+- bnc#897129
+  - protect imported data agains overwriting by read
```

Comment:
This description is too implementation oriented. We should rephrase it from the user's POV.